src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptState.java

```diff
@@ -31,20 +31,29 @@
     final MessageConsumer acceptReply;
     private final MessageConsumer initialThrottle;
     final Consumer<MessageConsumer> setThrottle;
+    final Consumer<Runnable> acceptReplyCleanup;
```

Comment:
Rename to `setCleanupConnectReply`, right?

Reply:
yep, makes sense.